### DISCUSSION TITLE
It would be better if you give option to edit readme md on different theme  - Dark, Light

### DISCUSSION THREAD

**Discussion by Vishal-beep136** [community/community#18924] [Accessibility] [answered]
I am wondering if you guys can implement this feature on Readme.Md

where developer can set readme image or content on depending Theme

i.e. - some dark logo not look good on dark mode

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Vishal-beep136 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Comment by cehfisher** [comment, marked as answer]
Hi @Vishal-beep136 - we wanted to follow up on this issue. Have the recent theme changes resolved this issue for you? If yes, could you please close out this issue? If not, what additional changes are needed? Please supply detailed instructions to reproduce your issue and include screenshots and/or URLs, if applicable. Thank you! (cc @queenofcorgis)
